### PR TITLE
LPS-201022 Replace AUI() usages in commerce-checkout-web

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-201022)

## Steps 

- Go to Control Panel > Sites > Create a Minium site
- In the Home page > upper side > click on Account > create a new account 
- In the left nav bar > select Catalog
- Add some items to the cart
- In the left nav bar > click on Account Management > edit the account
- Go to Addresses and and an address
- In the left nav bar > click Pending Orders 
- View the pending order and click checkout 
- When clicking on "Add Address Line", address 2 and 3 inputs should appear 
- When changing "CHOOSE SHIPPING ADDRESS" input, if we choose an address it should fill the inputs and disable them, and if we choose Add a New Address, the inputs should enable and be reseted.

https://github.com/liferay-echo/liferay-portal/assets/91208451/5db07545-2be0-453d-82be-fb2da50ae53a

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp [hunk 1]
```diff
@@ -320,52 +334,45 @@ boolean hasManageAddressesPermission = baseAddressCheckoutStepDisplayContext.has
...
-						);
-						commerceRegionIdSelect.parentElement.classList.add('d-none');
+						const selectedOption =
+							commerceAddress.options[commerceAddress.selectedIndex];
 
-						commerceRegionIdInput.value = selectedOption.getData('region');
-						commerceRegionIdInput.setAttribute(
-							'name',
-							'<portlet:namespace />regionId'
+						city.value = selectedOption.getAttribute('data-city');
```
Comment (p2kmgcl) [inline modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp:340]: Better use the dataset API here: `selectedOption.dataset.city`

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset

**Review by p2kmgcl — CHANGES_REQUESTED** [review]

**Comment by DiegoHu97** [comment]
@p2kmgcl Updated! I will notify commerce team about the changes 😄

**Review by luca-pellizzon — APPROVED** [review]

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
The results of ci:test:commerce-functional suite are good.

cc @victoroscuro

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/143595

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**